aiohttp/test_utils.py

```diff
@@ -413,7 +413,7 @@ def unittest_run_loop(func, *args, **kwargs):
     """
 
     @functools.wraps(func, *args, **kwargs)
-    def new_func(self):
+    def new_func(self, *args, **kwargs):
```

Comment:
Could you rename these params to `inner_args`/`inner_kwargs` or `new_args`/`new_kwargs` maybe?
Otherwise people will always stuck with this code.